### ISSUE TITLE
Request to speed up calibration when using long exposure time for guiding

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenPHDGuiding/phd2#404]
```
Request from Chris:

https://groups.google.com/d/topic/open-phd-guiding/pcbJ1n-kjzg/discussion

Having a long guide exposure can help with guiding, but is not necessary for 
calibration.  It would be nice to have a way to automatically decrease the 
guide exposure for calibration only.

The tricky part will be to find a way to do this without complicating the UI.

```

Original issue reported on code.google.com by `andy.gal...@gmail.com` on 22 Feb 2015 at 5:32

**Comment by agalasso** [comment]
related issue: #725

**Comment by bwdev01** [comment]
This has probably been handled by a combined use of auto-exposure and the variable-delay featured implemented in 2.6.11dev1.
